lighthouse-cli/bin.ts

```diff
@@ -229,11 +234,21 @@ function lighthouseRun(addresses: Array<string>, config: Object, flags: Object)
   return lighthouse(address, flags, config)
     .then((results: Results) => Printer.write(results, outputMode, outputPath))
     .then((results: Results) => {
+      const filename = `./${assetSaver.getFilenamePrefix({url: address})}.report.html`;
+
       if (outputMode === Printer.OutputMode[Printer.OutputMode.pretty]) {
-        const filename = `./${assetSaver.getFilenamePrefix({url: address})}.report.html`;
         Printer.write(results, 'html', filename);
       }
 
+      // Generate report.html, host it and open it in the default browser
+      if (flags.view) {
+        server.listen().then((port: number) => {
+          const fileAbsPath = `${__dirname}/server/reports/${filename}`;
+          Printer.write(results, 'html', fileAbsPath);
```

Comment:
We are already saving the file to disk in the default `pretty` case. Shall we just reuse that file and host that?